functions.yml

```diff
@@ -231,15 +241,15 @@ etl:
           doc: Alternate alleles
```

Comment:
For above: replace `The current contigName` with `The current contig name`. Replace `Reference allele` with `The reference allele`. Here, replace `Alternate alleles` with `An array of alternate alleles`

Reply:
Fixed